schemas/2019-09-16/Microsoft.HealthcareApis.json

```diff
@@ -22,6 +22,7 @@
         "apiVersion": {
           "type": "string",
           "enum": [
+            "2018-08-20-preview",
```

Comment:
Please create a separate file for each individual API version

Reply:
Removed reference to old version